### PR TITLE
Add redact API definition

### PR DESCRIPTION
**Pull request by mheap**
## Description

Adds redaction API reference

## Deploy Notes

Don't merge yet. This PR has been raised to provide product with an endpoint to review the docs

### CODE REVIEW

File: app/constraints/open_api_constraint.rb [hunk 1]
```diff
@@ -3,6 +3,7 @@
   media
   number-insight
   stitch
+  redact
```
Comment (tommorris) [inline app/constraints/open_api_constraint.rb]: I know this isn't ready to merge, but is Redact going to be the name and not Redaction API? 🤔

Comment (mheap) [reply app/constraints/open_api_constraint.rb]: I'm not sure. There's ongoing work to use the same product names across all systems, so I guess the answer will come out of that.

@nzmark - any input?

File: _documentation/redaction/overview.md [hunk 2]
```diff
@@ -0,0 +1,93 @@
+---
+title: Overview
+---
+
+# Overview
+
+As part of our General Data Protection Regulation (GDPR) program, Nexmo provide an API that allows you to manage personally identifiable information within the Nexmo platform. The redaction API allows you to redact information on demand, providing a solution for your own compliance needs.
```
Comment (tbedford) [inline _documentation/redaction/overview.md]: 'redaction API' to 'Redaction API'

File: _documentation/redaction/overview.md [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+
+To use the redaction API:
+
+1. Make a request to a Nexmo API. Save the ID returned in the response in your database for later reference
+2. When you receive a right to erasure request, fetch all IDs associated with that user from your database
+3. For each ID, make a request to the [Nexmo redaction API](/api/redaction)
+
+### Product API
+
+Some Nexmo APIs are not covered by the automatic redaction API as the data stored is controlled by the customer. These cases are detailed below
```
Comment (tbedford) [inline _documentation/redaction/overview.md]: 'automatic redaction API' - has this already been described?

**Comment by mheap** [comment]
This was reviewed by @tbedford

**Pull request merged**